### DISCUSSION TITLE
What is auto-parallel training  ?

### DISCUSSION THREAD

**Discussion by GxjGit** [hpcaitech/ColossalAI#2057] [Community | Q&A]
What is the specific function of automatic parallel connection? Is there any related detailed introduction or use demo?

![image](https://user-images.githubusercontent.com/24288375/204961133-8373729a-e3a8-4131-81e1-a95a395d37ae.png)

**Comment by binmakeswell** [comment]
Hi @GxjGit You can refer to
https://arxiv.org/abs/2302.02599
https://github.com/hpcaitech/ColossalAI/tree/main/examples/language/gpt/experiments/auto_parallel
https://github.com/hpcaitech/ColossalAI/tree/main/examples/tutorial/auto_parallel
